index.js

```diff
@@ -22,17 +22,22 @@ module.exports = function(req, res, next) {
 			for(var key in req.files) {
 				if (req.files.hasOwnProperty(key)) {
 					file = req.files[key];
-					if (err && (options && !options.reapOnError)) {
-						debug('skipped auto removal of %s - please manually deprecate.',  file.path);
-						continue;
+					if (!Array.isArray(file)) {
```

Comment:
I would suggest to use `!(file instanceof Array)` instead for compatibility reasons.